packages/manager/apps/dedicated/client/app/license/license.controller.js

```diff
@@ -171,7 +176,28 @@ export default /* @ngInject */ (
     $state.go('app.license.order');
   };
 
+  const checkCpanel = () => {
+    ovhFeatureFlipping
+      .checkFeatureAvailability(['license:cpanel-eol-banner'])
+      .then((availability) => {
+        if (!availability.isFeatureAvailable('license:cpanel-eol-banner'))
+          return;
+        $scope
+          .iceberg(`/license/cpanel`)
+          .query()
+          .execute()
+          .$promise.then(({ data }) => {
+            $scope.centos7Cpanel = data.length > 0;
```

Comment:
**nitpick:** Here you're checking if the user has any `cpanel` license. So `centos7Cpanel` does not seem like the appropriate variable name.